site/content/faqs/_index.md

```diff
@@ -1,80 +1,5 @@
-
-- **How do I set up aggregation in the coordinator?**
-Refer to the [Aggregation section](/docs/how_to/query) of the M3Query how-to guide.
-
-- **How do I set up aggregation using a separate aggregation tier?**
-See this [WIP documentation](https://github.com/m3db/m3/pull/1741/files#diff-0a1009f86783ca8fd4499418e556c6f5).
-
-- **Can you delete metrics from M3DB?**
-Not yet, but that functionality is currently being worked on.
-
-- **How can you tell if a node is snapshotting?**
-You can check if your nodes are snapshotting by looking at the `Background tasks` tab in the [M3DB Grafana dashboard](https://grafana.com/dashboards/8126).
-
-- **How do you list all available API endpoints?**
-See [M3DB OpenAPI](https://m3db.io/openapi).
-
-- **What is the recommended way to upgrade my M3 stack?**
-See the [Upgrading M3](/docs/operational_guide/upgrading_m3) guide.
-
-- **When graphing my Prometheus data in Grafana, I see gaps. How do I resolve this?**
-This is due to M3 having a concept of `null` datapoints whereas Prometheus does not. To resolve this, change `Stacking & Null value` to `Connected` under the `Visualization` tab of your graph.
-
-- **I am receiving the error `"could not create etcd watch","error":"etcd watch create timed out after 10s for key: _sd.placement/default_env/m3db"`**
-This is due to the fact that M3DB, M3Coordinator, etc. could not connect to the `etcd` server. Make sure that the endpoints listed under in the following config section are correct AND the correct configuration file is being used.
-```yaml
-etcdClusters:
-  - zone: embedded
-    endpoints:
-      - HOST1_STATIC_IP_ADDRESS:2379
-      - HOST2_STATIC_IP_ADDRESS:2379
-      - HOST3_STATIC_IP_ADDRESS:2379
-``` 
-
-- **How can I get a heap dump, cpu profile, etc.**
-See our docs on the [/debug/dump api](/docs/troubleshooting)
-
-- **How much memory utilization should I run M3DB at?**
-We recommend not going above 50%.
-
-- **What is the recommended hardware to run on?**
-TBA
-
-- **What is the recommended way to create a new namespace?**
-Refer to the [Namespace configuration guide](/docs/operational_guide/namespace_configuration).
-
-- **How can I see the cardinality of my metrics?**
-Currently, the best way is to go to the [M3DB Node Details Dashboard](https://grafana.com/grafana/dashboards/8126) and look at the `Ticking` panel. However, this is not entirely accurate because of the way data is stored in M3DB -- time series are stored inside time-based blocks that you configure. In actuality, the `Ticking` graph shows you how many unique series there are for the most recent block that has persisted. In the future, we plan to introduce easier ways to determine the number of unique time series. 
++++
+title = "FAQs"
+weight = 9
+chapter = true
++++
```

Comment:
Is there some purpose to leave this there?

Reply:
Yes, these are what Hugo calls leaf pages, and you need them to create sections, but they end up rendered as listings pages, like… https://deploy-preview-3029--m3db.netlify.app/docs/faqs/

But most people never see them, but Hugo insists on generating them